src/Protocol/ActivityPub/Receiver.php

```diff
@@ -644,16 +644,19 @@ public static function processActivity(array $activity, string $body = '', int $
 				return true;
 			}
 		} else {
-			$attributed_to = '';
+			$attributed_to = null;
 		}
 
 		// Test the provided signatures against the actor and "attributedTo"
 		if ($trust_source) {
-			if ($attributed_to !== false && $attributed_to !== '') {
```

Comment:
can we learn something about my recent discussion with @art4 about "loose checks" and "more robust and precise checks" found here https://github.com/friendica/friendica/pull/14913#discussion_r2095000214 ?

why do we need to explicitly set a string variable to null, just to check it later with a loose check? Wouldn't it be better to have a meaningful flag instead?

Reply:
The variable `$attributed_to` is fetched from the ActivityPub message. And when the message doesn't contain the variable `attributedTo`, then the variable `$attributed_to` is set to `null` - which is totally correct, I think.